collector/types/types.go

```diff
@@ -92,6 +92,23 @@ func GetHubWhitelistedValidatorsPair() (HubWhitelistedValidatorsRequest, HubWhit
 	return HubWhitelistedValidatorsRequest{}, HubWhitelistedValidatorsResponse{}
 }
 
+type ValidatorRegistryValidatorsRequest struct {
+	WhitelistedValidators struct{} `json:"get_validators_for_delegation"`
+}
+
+type Validator struct {
+	Active          bool   `json:"active"`
+	Address         string `json:"address"`
+	DelegationLimit string `json:"delegation_limit"` //uint128
```

Comment:
There are no fields such `delegation_limit` and `active`

Reply:
my bad. Struct based on schema/validator.json file, which seems outdated now.
type fixed